lib/daru/io/exporters/csv.rb

```diff
@@ -5,16 +5,28 @@ module IO
     module Exporters
       module CSV
```

Comment:
@zverok - Can you please confirm about whether we should refractor now, or can we keep the Module-function approach for now and refractor later?